content/en/real_user_monitoring/guide/best-practices-for-rum-sampling.md

```diff
@@ -0,0 +1,64 @@
+Sampling in Datadog's Real User Monitoring product enables you to collect data from a certain percentage of user traffic.
+
+This guide walks you through various best practices for RUM sampling so that you capture the sessions and collect data based on your monitoring needs.
+
+## Sampling configuration
+
+### Configure the variable names that correspond with the SDK version
+
+Sessions are randomly sampled based on the percentage listed in the [SDK configuration][1]. To that end, make sure to use the correct configuration variable names for the SDK version that is being used.
+
+### Configure the sampling rate
+Before each new user session, the SDK draws a random floating-point number between 0 and 1, which is then compared to the value that is set in the SDK configuration. If the random number is lower than the value set in the SDK configuration, the session is kept and events start being collected. If the value is higher, the session is not kept and events do not get collected at all until the end of the session.
+
+You can set the sampling rate via the SDK ([Browser][2], [Android][3], [iOS][4], [React Native][5], [Flutter][6], [Roku][7]), then deploy it in the application code.
+
+Only sessions that are sampled are available in RUM. For example, if the sampling rate is set to 60%, then 60% of all sessions and metrics (such as Core Web Vitals and usage numbers) are visible in RUM.
+
+The random sampling is by session, not by user.
+
+### The effect of sampling on data and metrics that are available in RUM
+RUM metrics (such as Core Web Vitals and usage numbers) are calculated based on sessions that are sampled. For example, if the sampling rate is set to capture 60% of sessions, then the Core Web Vitals and total number of sessions are calculated based on 60% of those sessions. 
+
+### Recommended sampling rate
+In terms of setting an ideal sampling rate, it depends on the amount of traffic you see and the data you are looking for. Datadog recommends starting with a sampling rate you are comfortable with based on your budget and estimated traffic, then tweak it based on the data you need. Learn more about how [sessions are defined][9] in RUM.
+
+### Sampling based on specific attributes
+Configuring sampling based on specific attributes, such as sampling 100% of sessions with errors and 5% otherwise, or only sampling sessions that go through the checkout flow, is not supported. If this is a feature request that is critical for your business needs, create a ticket with [Datadog Support][8].
+
+### Changing the sampling rate in the Datadog RUM UI
+Changing the sampling rate in the Datadog RUM UI is not supported. If this is a feature request that is critical for your business needs, create a ticket with [Datadog Support][8].
+
+### Adjusting sampling during live outages
+
+If a bug or incident occurs, you can increase sampling to collect 100% of your user traffic to ensure no session is missed. You need to deploy a code change to achieve this.
```

Comment:
This is not doable on Mobile and Roku due to the long release cycle and adoption rate.

Reply:
@plousada I updated the text. Does it work?